app/facades/foods_facade.rb

```diff
@@ -0,0 +1,12 @@
+class FoodsFacade
+  def self.top_10_results(string)
+    foods_list_data = FoodsService.get_foods(string)
+    foods_list_data[:foods].first(10).map do |food|
+      Food.new(food)
+    end
+  end
```

Comment:
I think this method could return the total count, and the top 10 food item results:
```
def search_results(string)
  foods_list_data = FoodsService.get_foods(string)
  foods = foods_list_data[:foods].first(10).map do |food|
    Food.new(food)
  end
  [foods, food_list_data[:totalHits]]
end
```
This way, you'd only make one call to the API, and you could get rid of the `foods_count` method in this facade.